### ISSUE TITLE
Heightmap

### ISSUE THREAD

**Issue by mohammad787** [Sienci-Labs/gsender#491]
Would you please add height mapping to the app

**Comment by J-eremy** [comment]
I second this, I know it was mentioned a while ago and there was even a menu for it at one time, and I also understand how much work it would be but this sentiment is also on my list. lol

**Comment by patofoto** [comment]
Yes!!!. Gender would be perfect if we could build surface meshes. I need this for PCB milling.

**Comment by apatieh** [comment]
I used to use candle but I'm using gsender for a while as it improved so much but it lacks this feature.
please add it

**Comment by SienciLabs** [comment]
Closed as duplicate of https://github.com/Sienci-Labs/gsender/issues/74

Technically this one came sooner, but the other has since had more conversation in it
